src/commandDetails/miscellaneous/member.ts

```diff
@@ -24,11 +24,20 @@ type UwIdType = string | undefined;
  */
 const getMemberEmbed = async (uwid: UwIdType): Promise<EmbedBuilder> => {
   const title = 'CSC Membership Information';
+  const EXPLAIN_MEMBERSHIP = `Oops! You didn't provide a UW ID.
+
+  If you're already a CSC member, please provide a UW ID.
```

Comment:
Instead of saying that if you are a member provide ID, the message should be something like this - 
`Please provide a UW ID to check if you are a CSC member. If you are not, are you interested in being one ...`

Reply:
Hi, I've updated the description of `EXPLAIN_MEMBERSHIP`. I also took care of linting for the `EmbedBuilder`s and reorganized the description constants to make it a bit more organized.